test/views/sections/dispatch/DispatchWarehouseExciseViewSpec.scala

```diff
@@ -55,6 +58,54 @@ class DispatchWarehouseExciseViewSpec extends SpecBase with ViewBehaviours {
           Selectors.link(1) -> messagesForLanguage.returnToDraft
         ))
       }
+
+      s"when ERN needs updating and there is NO form error '${messagesForLanguage.lang.code}'" - {
+
+        implicit val msgs: Messages = messages(Seq(messagesForLanguage.lang))
+
+
+        implicit val request: DataRequest[AnyContentAsEmpty.type] = dataRequest(FakeRequest(),
+          emptyUserAnswers.copy(submissionFailures =
+            DispatchWarehouseExcisePage.possibleErrors.map(error => dispatchWarehouseExciseInvalid.copy(error.code))
+          )
+        )
+
+        lazy val view = app.injector.instanceOf[DispatchWarehouseExciseView]
```

Comment:
I would move this into the class scope (i.e. above `"DispatchWarehouseExciseView" - {`) so you don't have to instantiate 2 views